lib/commands/project/project.js

```diff
@@ -378,11 +381,15 @@ function spawn(command, options, cwd, cb) {
 //.option('-p, --path-name [path]', 'a sepecific path of the api')
 //.option('-f, --test-module <module>', 'one of: ' + testmodules)
 //.option('-t, --assertion-format <type>', 'one of: ' + assertiontypes)
+//.option('-o, --force', 'allow overwriting of all existing test files matching those generated')
 function testGenerate(directory, options, cb) {
   var pathList = [];
   var desiredPaths = [];
   var testModule = options.testModule || TESTMODULES[0];
   var assertionFormat = options.assertionFormat || ASSERTIONTYPES[0];
+  var overwriteAll = options.force || false;
+  var conflicts;
```

Comment:
Unless this variable should be accessible from here, do not declare variables here.